types/version.ts

```diff
@@ -0,0 +1 @@
+export const FRONTEND_APP_VERSION = '0.1.3'
```

Comment:
Hi @cbh778899 , thanks you. I think here it shouldn't be a `const`, every time we add new features or fix issues. We need to bump this version.

Reply:
that's true, but still we want to hard-code the version instead use a dynamic one, so if we want to change the version just change it here...or there's another way to query from env vars, but that possibly can cause extra effort on each deloy?

Reply:
For the backend, we set version in configuration https://github.com/SkywardAI/chat-backend/blob/0321b7946d0dbbde1e0a692986a923c0e8a9d1c9/backend/src/config/settings/base.py#L13.

What is your idea?

Reply:
Good idea, I can add a configure.ts something like that, let me add another commit

Reply:
This should be okay now, I've set it in .env file